src/main/java/org/jenkinsci/plugins/gitclient/GitUsernamePasswordBinding.java

```diff
@@ -0,0 +1,162 @@
+package org.jenkinsci.plugins.gitclient;
+
+import com.cloudbees.plugins.credentials.common.StandardCredentials;
+import com.cloudbees.plugins.credentials.common.StandardUsernamePasswordCredentials;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import hudson.Extension;
+import hudson.EnvVars;
+import hudson.FilePath;
+import hudson.Functions;
+import hudson.Launcher;
+import hudson.model.Run;
+import hudson.model.TaskListener;
+
+import org.apache.commons.lang.StringUtils;
+
+import org.jenkinsci.Symbol;
+import org.jenkinsci.plugins.credentialsbinding.BindingDescriptor;
+import org.jenkinsci.plugins.credentialsbinding.MultiBinding;
+import org.jenkinsci.plugins.credentialsbinding.impl.AbstractOnDiskBinding;
+import org.jenkinsci.plugins.credentialsbinding.impl.UnbindableDir;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
```

Comment:
Need to be removed.  The `javax.annotation` classes are deprecated in Jenkins.  We should use the findbugs annotations for `NonNull` instead.
```suggestion
import javax.annotation.Nonnull;
import javax.annotation.Nullable;
```

Reply:
Things got messy after merging the test branch I was working on in the `gitUsernamePassword` branch. But lesson learn will be more care full next time. Making things right in the `gitUsernamePassword` branch and will make a well verified commit this time, my apologizes.